### PR TITLE
kevin: migration for new build host

### PR DESCRIPTION
**Pull request by TheJJ**
[Kevin](https://github.com/SFTtech/kevin) builds are migrated from within a KVM VM to a podman container within a KVM VM.

### CODE REVIEW

File: kevinfile [hunk 1]
```diff
@@ -5,17 +5,16 @@
...
 # Make sure both release and debug are tested. Arch job has more checks,
 # so it should run debug, while Debian can test build in release.
 configure:
-	- env: mode=release compiler=gcc            (? if job == "debian" ?)
-	- env: mode=debug   compiler=gcc            (? if job == "arch-gcc" ?)
-	- env: mode=debug   compiler=clang          (? if job == "arch-clang" ?)
-	./configure --mode=${mode} --compiler=${compiler} --flags="-Werror" --ccache --download-nyan
+	- env: mode=debug   compiler=gcc            (? if job == "debian" ?)
+	- env: mode=debug   compiler=clang          (? if job == "debian-clang" ?)
+	./configure --mode=${mode} --compiler=${compiler} --ccache --download-nyan
```
Comment (simonsan) [inline kevinfile:17]: Small reminder, to not forget: `--flags="-Werror"` we should probably put that back in before merging. ;-)

**Comment by simonsan** [comment]
2020-05-10 05_57_46- WIP  Buildsystem update and documentation by simonsan · Pull Request #1266 · SF-Screenshot

I think Kevin is doing something wrong there It seems it's not updating the remote state and links somehow? If I click on the link to the check in the downwards panel I come to an old check. I need to click on the result of the commit and go from there.

**Comment by TheJJ** [comment]
API-wise it's the same, kevin updates the status for a commit hash. I'd try reloading the page, maybe that helps?

**Comment by simonsan** [comment]
> API-wise it's the same, kevin updates the status for a commit hash. I'd try reloading the page, maybe that helps?

No :-( but that's what it was like before. Reloading the page was updating the checks, when it was hanging. Now not anymore. :-(

**Comment by TheJJ** [comment]
Should be good for now, I've removed the werror and we'll add it (just for the CI!) once I resolved the cython-c++-warnings.

**Pull request merged**